src/content/translations/cs/javascript/index.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Ethereum pro vývojáře v JavaScriptu
-description: Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScriptu
+title: Ethereum pro vývojáře v JavaScript
```

Comment:
I suspect we may want to leave translations alone - these may be intentional changes. What do you think @lukassim @minimalsm?

Reply:
```suggestion
title: Ethereum pro vývojáře v JavaScriptu
```

Reply:
Agreed, most languages have case endings to indicate a grammatical case, so we should be careful to retain these.